File: Module 3/31_Arrays/Java/Arrays.java
public class Arrays {
    public static void main(String[] args) {
        int[] x = {5, 4, 3, 2, 1};

        int length = x.length;

        System.out.println("Accessing elements in the array:");
        for (int i = 0; i < length; i++) {
            System.out.print(x[i] + " ");
        }
    }
}

File: Module 3/31_Arrays/Cpp/Arrays.cpp
#include <iostream>
using namespace std;

int main() {
    int x[] = {5, 4, 3, 2, 1};

    int length = sizeof(x) / sizeof(x[0]);

    cout << "Accessing elements in the array:\n";
    for (int i = 0; i < length; i++) {
        cout << *(x + i) << " ";
    }

    return 0;
}
